src/components/theme/styles.js

```diff
@@ -64,6 +64,6 @@ export const theme = {
   feedItems: {
     borderRadius: 8,
     itemBackgroundColor: '#fff',
-    height: 84,
+    height: normalize(84),
```

Comment:
why are we _normalizing_ the height?

Reply:
because fonts are resized with normalize, and in different devices, mainly big devices, there is not enough space with 84px, so I need to normalize it to make it fit the content inside it

Reply:
neat :ok_hand: